### ISSUE TITLE
CurlException when trying to fetch products

### ISSUE THREAD

**Issue by MrMooky** [jackabox/statamic-shopify#93]
Disclaimer: I have not purchased the add-on yet as I wanted to try it out to make sure it works as expected and fits our use-case.

--

Installing went fine, everything is set up within Shopify and Statamic is running. But when I want to fetch products, I get the following error:

```
[2021-10-18 09:46:50] live.ERROR: 3 :  {"userId":"a25f2d67-72f5-4fd5-af3e-4f39cd3466ab","exception":"[object] (PHPShopify\\Exception\\CurlException(code: 0): 3 :  at /home/webapps/shopify-test/releases/20211018094131/vendor/phpclassic/php-shopify/lib/CurlRequest.php:200)
[stacktrace]
#0 /home/webapps/shopify-test/releases/20211018094131/vendor/phpclassic/php-shopify/lib/CurlRequest.php(95): PHPShopify\\CurlRequest::processRequest()
#1 /home/webapps/shopify-test/releases/20211018094131/vendor/phpclassic/php-shopify/lib/HttpRequestJson.php(71): PHPShopify\\CurlRequest::get()
#2 /home/webapps/shopify-test/releases/20211018094131/vendor/phpclassic/php-shopify/lib/ShopifyResource.php(343): PHPShopify\\HttpRequestJson::get()
#3 /home/webapps/shopify-test/releases/20211018094131/vendor/jackabox/statamic-shopify/src/Traits/FetchAllProducts.php(15): PHPShopify\\ShopifyResource->get()
#4 /home/webapps/shopify-test/releases/20211018094131/vendor/jackabox/statamic-shopify/src/Http/Controllers/CP/ImportProductsController.php(19): Jackabox\\Shopify\\Http\\Controllers\\CP\\ImportProductsController->fetchProducts()
```

Is this because the add-on is not purchased yet, or related to something else? Tried both locally and on a production server.

**Comment by MrMooky** [comment]
@jackabox Yes, I followed the instructions.

Apparently, the url I entered was wrong. I used the one that was showing in the Shopify backend under "Example URL". So the Curl error is gone now and the import says "Import has been queued." but then nothing is imported.

There is one product added in the shop: https://stella-projects-x-tireh-demo.myshopify.com/products/product-one.

Also, when I open the "Create Entry" page for the Products collection, I get this error:

<img width="611" alt="Screenshot 2021-10-19 at 07 57 34" src="https://user-images.githubusercontent.com/3603346/137852107-adf3fd4d-8b72-44fb-8077-2bc673c9cd73.png">

Nothing is written into the actual log.
